src/spacewar/ui/complex.cljc

```diff
@@ -1,17 +1,17 @@
 (ns spacewar.ui.complex
-  (:require [quil.core :as q]
-            (spacewar.ui [protocols :as p]
-                         [view-frame :as f]
-                         [config :as uic])
-            (spacewar.ui.control-panels [scan-panel :refer [->scan-panel]]
-                                        [engine-panel :refer [->engine-panel]]
-                                        [weapons-panel :refer [->weapons-panel]]
-                                        [damage-panel :refer [->damage-panel]]
-                                        [status-panel :refer [->status-panel]]
-                                        [deploy-panel :refer [->deploy-panel]])
-            (spacewar.ui.widgets [lights :refer [->indicator-light
-                                                 rectangle-light
-                                                 round-light]])))
+  (:require [quil.core :as q #?@(:cljs [:include-macros true])]
+            [spacewar.ui.protocols :as p]
```

Comment:
Just like ClojureScript disallows `:refer :all`, it also disallows prefix lists in require specs. (This is covered here in the "Differences from Clojure" page https://clojurescript.org/about/differences#_namespaces)